application/src/main/java/org/togetherjava/tjbot/features/help/AutoPruneHelperRoutine.java

```diff
@@ -163,7 +163,8 @@ private void pruneMemberFromRole(Member member, Role role, ForumChannel helpForu
         guild.removeRoleFromMember(member, role)
             .flatMap(any -> member.getUser().openPrivateChannel())
             .flatMap(channel -> channel.sendMessage(dmMessage))
-            .queue();
+            .queue(null,
+                    failure -> logger.debug("Tried to send a DM to a user but failed: ", failure));
```

Comment:
the log message is missing context. should add sth like
`Failed sending a DM to user (1234567) while pruning them from a helper role.` (with the number being the user ID)

Reply:
Done! 🙌